module/Entra/Microsoft.Entra/Utilities/Get-EntraUserCertificateUserIdsFromCertificate.ps1

```diff
@@ -0,0 +1,223 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+
+<#
```

Comment:
Remove these docs. We use docs in the md files

Reply:
removed